app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -112,4 +161,8 @@ class StateFragmentPresenter @Inject constructor(
     }
     return ephemeralStateResult.getOrDefault(EphemeralState.getDefaultInstance())
   }
+
+  private fun dpToPx(dp: Int): Int {
```

Comment:
Prefer using https://developer.android.com/reference/android/content/res/Resources.html#getDimensionPixelSize(int) with a dimension resource for the 8dp. That way the value is XML-driven and configurable.

Reply:
Removed